pull_request_procedure.md

```diff
@@ -9,8 +9,9 @@ When opening a Pull request, there is a checklist of points that appears on the
 - [ ] My code follows the style guidelines of this project
 - [ ] I have merged the develop branch into my branch and fixed any merge conflicts
 - [ ] I have performed a self-review of my own code
+- [ ] I have commented my code in hard-to-understand areas
 - [ ] My changes generate no new warnings
-- [ ] New and existing unit tests pass locally with my changes
+- [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] I have tested new or existing tests and made sure that they pass
```

Comment:
using "or" on the "new or existing" part of the sentence makes it sounds like you should choose one or the other